src/templates/ssr/fabric/index.svelte

```diff
@@ -0,0 +1,154 @@
+<!-- https://polyfill.io/v3/polyfill.min.js?features=default -->
+<script lang="ts">
+	import { CLICK_MACRO, GAMVariable } from '$lib/gam';
+	import { isValidReplacedVariable } from '$lib/gam';
+	import AdvertisementLabel from '$templates/components/AdvertisementLabel.svelte';
+	import Pixel from '$templates/components/Pixel.svelte';
+
+	export let Trackingpixel: GAMVariable;
+	export let Researchpixel: GAMVariable;
+	export let Viewabilitypixel: GAMVariable;
+	export let ClickthroughUrl: GAMVariable;
+	export let thirdPartyJSTracking: GAMVariable;
+
+	export let Layer1BackgroundImage: GAMVariable;
+	export let Layer1BackgroundPosition: GAMVariable;
+	export let Layer2BackgroundImage: GAMVariable;
+	export let Layer2BackgroundPosition: GAMVariable;
+	export let Layer3BackgroundImage: GAMVariable;
+	export let Layer3BackgroundPosition: GAMVariable;
+
+	export let MobileLayer1BackgroundImage: GAMVariable;
+	export let MobileLayer1BackgroundPosition: GAMVariable;
+	export let MobileLayer2BackgroundImage: GAMVariable;
+	export let MobileLayer2BackgroundPosition: GAMVariable;
+	export let MobileLayer3BackgroundImage: GAMVariable;
+	export let MobileLayer3BackgroundPosition: GAMVariable;
+</script>
+
+<AdvertisementLabel />
+<div class="creative creative--fabric">
+	<a
+		id="linkDesktop"
+		class="blink gs-container link hide-until-tablet"
+		href={`${CLICK_MACRO}${ClickthroughUrl}`}
+		target="_blank"
+	>
+		<div class="creative-container">
+			<div class="alt">
+				<div
+					class="layer layer1"
+					style="background-image: url('{Layer1BackgroundImage}');
+                        background-position: {Layer1BackgroundPosition}"
+				/>
+				<div
+					id="layer2"
+					class="layer layer2"
+					style="background-image: url('{Layer2BackgroundImage}');background-position: {Layer2BackgroundPosition}"
+				/>
+				<div
+					class="layer layer3"
+					style="background-image: url('{Layer3BackgroundImage}');
+                        background-position: {Layer3BackgroundPosition}"
+				/>
+			</div>
+		</div>
+	</a>
+	<a
+		id="linkMobile"
+		class="blink gs-container link mobile-only"
+		href={`${CLICK_MACRO}${ClickthroughUrl}`}
+		target="_blank"
+	>
+		<div class="creative-container">
+			<div class="alt">
+				<div
+					class="layer layer1"
+					style="background-image: url('{MobileLayer1BackgroundImage}');
```

Comment:
in svelte you can do `style:background-image={url('{MobileLayer1BackgroundImage}'}` if you prefer, same with background position

Reply:
I tried to do that as you suggested but there is an error with url `'url' is not defined`. Could you please expand on the syntax when you have more than one style and also why is there an error?

Reply:
Sorry I missed the quotes/backticks, should be ``style:background-image={`url('{MobileLayer1BackgroundImage}')`}``